apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp

```diff
@@ -155,7 +155,7 @@
                         <div class="ui divider hidden"></div>
 
                         <div class="align-right buttons">
-                            <a href="javascript:goBack()" class="ui button secondary">
+                            <a href='<%=request.getParameter("callback")%>' class="ui button secondary">
```

Comment:
Do we always get the `callback` param?
Shall we fallback to `javascript:goBack()` if it's not there?

Reply:
Also lets use double quotes and escape.
```suggestion
                            <a href="<%=request.getParameter(\"callback\")%>" class="ui button secondary">
```